types/src/account_address.rs

```diff
@@ -14,8 +14,9 @@ use std::{
     str::FromStr,
 };
 
-const SALT: &[u8] = b"aptos_framework::staking_contract";
-const VESTING_POOL_SALT: &[u8] = b"aptos_framework::vesting";
+const MULTISIG_ACCOUNT_DOMAIN_SEPARATOR: &[u8] = b"aptos_framework::multisig_account";
+const STAKING_CONTRACT_DOMAIN_SEPARATOR: &[u8] = b"aptos_framework::staking_contract";
+const VESTING_POOL_DOMAIN_SEPARATOR: &[u8] = b"aptos_framework::vesting";
```

Comment:
nits: ... 18 and 19 would be better in a separate PR...

Reply:
err those are minor and I'm too lazy to split them out 🤣